app.py

```diff
@@ -0,0 +1,127 @@
+"""
+DELETE THIS MODULE STRING AND REPLACE IT WITH A DESCRIPTION OF YOUR APP.
+
+app.py Template
+
+The app.py script does several things:
+- import the necessary code
+- create a subclass of ClamsApp that defines the metadata and provides a method to run the wrapped NLP tool
+- provide a way to run the code as a RESTful Flask service
+
+
+"""
+
+import argparse
+import logging
+
+# Imports needed for Clams and MMIF.
+# Non-NLP Clams applications will require AnnotationTypes
+
+from clams import ClamsApp, Restifier
+from mmif import Mmif, AnnotationTypes, DocumentTypes
+import pickle
+from sklearn.feature_extraction.text import CountVectorizer
+from sklearn.feature_extraction.text import TfidfTransformer
+
+# Import necessary function(s) from tfidf.py
+from tfidf import get_keywords
+
+class TfidfKeywordextractor(ClamsApp):
+
+    def __init__(self):
+        super().__init__()
+
+    def _appmetadata(self):
+        pass
+
+    def _annotate(self, mmif: Mmif, **parameters) -> Mmif:
+        self.mmif = mmif if type(mmif) is Mmif else Mmif(mmif)
+        idf_feature_file = parameters['idfFeatureFile']
+        topn = parameters['topN']
+        new_view = self._new_view(parameters)
+
+        # We assume the results from the text slicer is in the last view.
+        # If there is nothing in the last view, it means the input is a simple text document.
+        text_slicer_checker = self.mmif.views.get_last()
```

Comment:
This checker might not always work. Can you try to run this (short) pipeline and see if the result is expected? 
1. whisper https://apps.clams.ai/whisper-wrapper/v7/
2. tfidf